backend/connector-integration/src/connectors/payload/transformers.rs

```diff
@@ -0,0 +1,825 @@
+use std::collections::HashMap;
+
+use common_enums::enums;
+use common_utils::{
+    consts::{NO_ERROR_CODE, NO_ERROR_MESSAGE},
+    ext_traits::ValueExt,
+    types::StringMajorUnit,
+};
+use domain_types::{
+    connector_flow::{Authorize, Capture, RSync, Refund, SetupMandate, Void},
+    connector_types::{
+        MandateReference, PaymentFlowData, PaymentVoidData, PaymentsAuthorizeData,
+        PaymentsCaptureData, PaymentsResponseData, RefundFlowData, RefundSyncData, RefundsData,
+        RefundsResponseData, ResponseId, SetupMandateRequestData,
+    },
+    errors,
+    payment_method_data::{Card, PaymentMethodData, PaymentMethodDataTypes},
+    router_data::{
+        AdditionalPaymentMethodConnectorResponse, ConnectorAuthType, ConnectorResponseData,
+        ErrorResponse,
+    },
+    router_data_v2::RouterDataV2,
+};
+use error_stack::ResultExt;
+use hyperswitch_masking::{ExposeOptionInterface, PeekInterface, Secret};
+use serde::{Deserialize, Serialize};
+
+use super::{requests, responses};
+use crate::connectors::payload::{PayloadAmountConvertor, PayloadRouterData};
+use crate::types::ResponseRouterData;
+
+// Re-export types for use in main connector file
+pub use super::requests::{
+    PayloadCancelRequest, PayloadCaptureRequest, PayloadCardsRequestData, PayloadPaymentsRequest,
+    PayloadRefundRequest,
+};
+pub use super::responses::{
+    PayloadAuthorizeResponse, PayloadCaptureResponse, PayloadErrorResponse, PayloadEventDetails,
+    PayloadPSyncResponse, PayloadPaymentsResponse, PayloadRSyncResponse, PayloadRefundResponse,
+    PayloadSetupMandateResponse, PayloadVoidResponse, PayloadWebhookEvent, PayloadWebhooksTrigger,
+};
+
+type Error = error_stack::Report<errors::ConnectorError>;
+
+// Helper function to check if capture method is manual
+fn is_manual_capture(capture_method: Option<enums::CaptureMethod>) -> bool {
+    matches!(capture_method, Some(enums::CaptureMethod::Manual))
+}
+
+// Helper function to get card expiry in format "MM/YY"
+fn get_card_expiry<T: PaymentMethodDataTypes>(card: &Card<T>) -> Result<Secret<String>, Error> {
+    let month = card.card_exp_month.peek();
+    let year = card.card_exp_year.peek();
+
+    // Get last 2 digits of year
+    let year_2_digit = if year.len() >= 2 {
+        &year[year.len() - 2..]
+    } else {
+        year
+    };
+
+    Ok(Secret::new(format!("{}/{}", month, year_2_digit)))
+}
```

Comment:
Can we check if this function already exists in utils or not? If it's not present then can you move this function to utils file?